### PR TITLE
WIP building for dnx 1.0.0-beta8 coreclr on Windows

### PR DESCRIPTION
**Pull request by ctaggart**
I think I made good progress. I think most of it is building. See:
https://ci.appveyor.com/project/ctaggart/visualfsharp

I switched AppVeyor to install dnvm, then dnx, then restore with `dnu restore` instead of `nuget restore`. I upgraded the dependencies to be up-to-date.

### CODE REVIEW

**Comment by ctaggart** [comment]
I'm at a loss for what this means:
https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/build/860
image

I do not have `C:\Program Files (x86)\MSBuild\Microsoft\NuGet\Microsoft.NuGet.targets` on my computer.

File: src/FSharpSource.targets [hunk 1]
```diff
@@ -106,8 +106,8 @@
   <PropertyGroup>
     <NUnitVersion>2.6.4</NUnitVersion>
     <NUnitFullVersion>2.6.4.14350</NUnitFullVersion>
-    <NUnitLibDir>$(FSharpSourcesRoot)\..\packages\NUnit.$(NUnitVersion)\lib\</NUnitLibDir>
-    <NunitToolsLibDir>$(FSharpSourcesRoot)\..\packages\NUnit.Runners.$(NUnitVersion)\tools\lib\</NunitToolsLibDir>
+    <NUnitLibDir>$(FSharpSourcesRoot)\..\packages\NUnit\$(NUnitVersion)\lib\</NUnitLibDir>
+    <NunitToolsLibDir>$(FSharpSourcesRoot)\..\packages\NUnit.Runners\$(NUnitVersion)\tools\lib\</NunitToolsLibDir>
```
Comment (ctaggart) [inline src/FSharpSource.targets]: `dnu restore` is a bit different than `nuget restore` in its folder hierarchy.

**Comment by ctaggart** [comment]
Getting really close to having this work. Tests are passing. Looks like I have another nuget command to remove.
image

**Comment by KevinRansom** [comment]
@ctaggart  Yhanks for trying this out, however, we will be tracking builds of .NetCore from the .netcore OSS project.  We will not be targeting a specific packaging of it such as DNX 1.0.0-beta8 coreclr.

I hope that makes sense.

Kevin

**Comment by ctaggart** [comment]
@KevinRansom I don't understand. This is tracking builds from their oss project.

A bigger problem I'm having right now is that my local build and my appveyor build both get this error, but the PR for the same commit doesn't error out on your AppVeyor project. Do you have any environment variables set? Can you paste what you see here:

https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/settings/yaml

Here is the error I get on my AppVeyor:

image

which is the same error I see locally:

image

**Comment by ctaggart** [comment]
There appear to be image differences between Paid and Open Source accounts that are causing the difference. I opened up this support request:
[Differences in 'os: Windows Server 2012 R2' for Pro & Open Source](http://help.appveyor.com/discussions/problems/3291-differences-in-os-windows-server-2012-r2-for-pro-open-source)

**Comment by ctaggart** [comment]
I set the AppVeyor to `os: Visual Studio 2015` and now the error I'm seeing on my work computer, home computer, and on the open source AppVeyor builds, now shows up here too. It is the same error reported here 14 days ago: https://github.com/Microsoft/visualfsharp/issues/649#issuecomment-145370680

**Comment by ctaggart** [comment]
Locally, my current failure is `'tryHead' is not defined`.

```
"C:\fs\visualfsharp\tests\..\packages\NUnit.Runners\2.6.4\tools\\nunit-console.exe" /nologo /result=CoreUnit_net40_Xml.xml /output=CoreUnit_net40_Output.log /e
rr=CoreUnit_net40_Error.log /work=C:\fs\visualfsharp\tests\\TestResults C:\fs\visualfsharp\tests\..\release\net40\bin\..\..\net40\bin\FSharp.Core.Unittests.dll

ProcessModel: Default    DomainUsage: Single
Execution Runtime: net-3.5
...............................................................................................................................................................
...............................................................................................................................................................
...............................................................................................................................................................
...............................................................................................................................................................
...............................................................................................................................................................
...........................................................................................................................
Tests run: 918, Errors: 0, Failures: 0, Inconclusive: 0, Time: 148.098601463195 seconds
  Not run: 0, Invalid: 0, Ignored: 0, Skipped: 0

perl C:\fs\visualfsharp\tests\\fsharpqa\testenv\bin\runall.pl -resultsroot C:\fs\visualfsharp\tests\\TestResults -results FSharp_Results.log -log FSharp_Failur
es.log -fail FSharp_Failures -cleanup:yes -ttags:coreclr -nottags:ReqPP,NOOPEN,NO_CI,Expensive -procs:0
One or more output files exist from a previous test run and will be renamed:
        Renaming 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Results.log' to 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Results.log.04'
        Renaming 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Failures.log' to 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Failures.log.04'
        Renaming 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Failures.env' to 'C:\fs\visualfsharp\tests\\TestResults\FSharp_Failures.env.04'
fsharp\core\access -- failed
fsharp\core\apporder -- failed
fsharp\core\array -- failed
fsharp\core\csext -- failed
fsharp\core\genericmeasures -- failed
fsharp\core\innerpoly -- failed
fsharp\core\int32 -- failed
fsharp\core\lazy -- failed
fsharp\core\lift -- failed
fsharp\core\map -- failed
fsharp\core\math\numbers -- failed
fsharp\core\math\numbersVS2008 -- failed
fsharp\core\members\ctree -- failed
fsharp\core\members\factors -- failed
fsharp\core\members\ops -- failed
fsharp\core\namespaces -- failed
fsharp\core\nested -- failed
fsharp\core\printf -- failed
fsharp\core\reflect -- failed
fsharp\core\seq -- failed
fsharp\core\subtype -- failed
fsharp\core\tlr -- failed
fsharp\core\unicode -- failed


0 tests passed (0.00%)
23 tests failed (100.00%)
0 tests were cascaded failures (0.00%)
0 tests returned "no result" (0.00%)
0 tests timed out (0.00%)
0 tests had test errors (0.00%)
========
23 Total executed tests

0 tests were skipped
========
23 Total tests (executed & skipped)

41s Total time
+++ fsharp\core\access +++
calling build.bat in C:\fs\visualfsharp\tests\fsharp\core\access

 FSCOREDLL_CORECLR_PATH='C:\fs\visualfsharp\tests\..\release\coreclr\bin\fsharp.core.dll'
PERMUTATIONS='FSC_CORECLR'
---------------------------------------------------------------
Executables

ALINK               =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\\al.exe
CORDIR              =C:\Windows\MICROS~1.NET\FRAMEW~1\V40~1.303
CORSDK              =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\
CSC                 ="C:\Windows\MICROS~1.NET\FRAMEW~1\V40~1.303\csc.exe" /nologo
csc_flags           =/nologo
FSC                 =C:\fs\visualfsharp\tests\..\release\net40\bin\fsc.exe
fsc_flags           =  -r:System.Core.dll --nowarn:20 --define:COMPILED
FSCBinPath          =C:\fs\visualfsharp\tests\..\release\net40\bin
FSCOREDLL20PATH     =C:\Program Files (x86)\Reference Assemblies\Microsoft\FSharp\.NETFramework\v2.0\2.3.0.0\FSharp.Core.dll
FSCOREDLLPATH       =C:\fs\visualfsharp\tests\..\release\net40\bin\FSharp.Core.dll
FSCOREDLLPORTABLEPATH =C:\fs\visualfsharp\tests\..\release\net40\bin\..\..\portable47\bin\FSharp.Core.dll
FSCOREDLLNETCOREPATH=C:\fs\visualfsharp\tests\..\release\net40\bin\..\..\portable7\bin\FSharp.Core.dll
FSCOREDLLNETCORE78PATH=C:\fs\visualfsharp\tests\..\release\net40\bin\..\..\portable78\bin\FSharp.Core.dll
FSCOREDLLNETCORE259PATH=C:\fs\visualfsharp\tests\..\release\net40\bin\..\..\portable259\bin\FSharp.Core.dll
FSDATATPPATH        =C:\fs\visualfsharp\tests\..\release\net40\bin\FSharp.Data.TypeProviders.dll
FSDIFF              =C:\fs\visualfsharp\tests\fsharpqa\testenv\bin\diff.exe
FSI                 =C:\fs\visualfsharp\tests\..\release\net40\bin\fsiAnyCPU.exe
fsi_flags           =  -r:System.Core.dll --nowarn:20 --define:INTERACTIVE --maxerrors:1 --abortonerror
GACUTIL             =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\\gacutil.exe
ILDASM              =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\\ildasm.exe
INSTALL_SKU         =CLEAN
MSBUILDTOOLSPATH    =C:\Program Files (x86)\MSBuild\14.0\Bin\
NGEN                =C:\Windows\MICROS~1.NET\FRAMEW~1\V40~1.303\ngen.exe
PEVERIFY            =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\\peverify.exe
RESGEN              =C:\PROGRA~2\MICROS~1\Windows\v8.1A\bin\NETFX4~1.1TO\\resgen.exe
---------------------------------------------------------------
REDUCED_RUNTIME set
Build for CORECLR
 --exec C:\fs\visualfsharp\tests\fsharp\single-test-buildandDeploy.fsx --testPlatform:win7-x86 --source:"C:\fs\visualfsharp\tests\fsharp\coreclr_utilities.fs"
--source:" test.fsx"  --fscPath:C:\fs\visualfsharp\tests\fsharp\..\..\release\net40\bin\fsc.exe --packagesDir:C:\fs\visualfsharp\tests\fsharp\..\..\packages  -
-testProjectJson:C:\fs\visualfsharp\tests\fsharp\project.json --testProjectJsonLock:C:\fs\visualfsharp\tests\fsharp\project.lock.json --fsharpCore:C:\fs\visual
fsharp\tests\..\release\coreclr\bin\fsharp.core.dll --dnuPath:"C:\fs\visualfsharp\tests\fsharp\..\..\packages\dnx-coreclr-win-x86.1.0.0-beta6-12032\bin\dnu.cmd
" --nugetSources:https://www.myget.org/F/dotnet-core;https://www.myget.org/F/dotnet-corefx;https://www.myget.org/F/dotnet-buildtools --define:CoreClr --define:
NetCore --InitializeForTests:true --output:C:\fs\visualfsharp\tests\fsharp\..\testbin\release\coreclr\fsharp\core\access\output\test.exe --testDirectory:C:\fs\
visualfsharp\tests\fsharp\..\testbin\release\coreclr\fsharp\core\access\TestDirectory --compilerDirectory:C:\fs\visualfsharp\tests\fsharp\..\testbin\release\co
reclr\fsharp\core\access\Compiler --compilerJsonLock:"C:\fs\visualfsharp\tests\fsharp\..\..\src\fsharp\Fsc\project.lock.json"


single-test-buildandDeploy.fsx(16,136): error FS0039: The value, constructor, namespace or type 'tryHead' is not defined
Errorlevel: 1
Test Script Failed (perhaps test did not emit test.ok signal file?)
```

**Comment by ctaggart** [comment]
The build scripts just run `single-test-buildandDeploy.fsx` which uses the `fsi` on the `path`. I just had to make sure the `4.0` version was on the path to fix that.
image

Now I just need to find out why CoreRun.exe isn't showing up in the build directories. My hunch is that it will when I put back the `runtimes` in the `project.json` files.

**Comment by ctaggart** [comment]
How do I get `CoreRun.exe` to make it in the test directories? If you look at the last failed build, you will see this:
image

I get the same issue locally. It simply isn't in the folders and I'm not how it gets there exactly.
image

I was thinking that `tests\fsharp\project.json` would copy it there since it references `TestHost`.
image

image

I could copy the executable there. My understanding is that it must be in the same directory.
```
"C:\fs\visualfsharp\packages\runtime.win7-x86.Microsoft.NETCore.TestHost\1.0.0-beta-23413\runtimes\win7-x86\native\CoreRun.exe"
"C:\fs\visualfsharp\packages\runtime.win7-x64.Microsoft.NETCore.TestHost\1.0.0-beta-23413\runtimes\win7-x64\native\CoreRun.exe"
```

**Comment by ctaggart** [comment]
Looks like the `getNativeFiles` is not working with the updated dependencies. It needs to look for them in the `runtime.win7-x64`* structure now too.
https://github.com/ctaggart/visualfsharp/blob/dnx/tests/fsharp/single-test-buildandDeploy.fsx#L143-L181

**Comment by ctaggart** [comment]
I fixed `getNativeFiles` so that it gets files so that it gets the files from these paths on my system:
```
C:\fs\visualfsharp\tests\fsharp\..\..\packages\runtime.win7-x86.Microsoft.NETCore.Runtime.CoreCLR\1.0.1-beta-23413
C:\fs\visualfsharp\tests\fsharp\..\..\packages\runtime.win7-x86.Microsoft.NETCore.ConsoleHost\1.0.0-beta-23413
C:\fs\visualfsharp\tests\fsharp\..\..\packages\runtime.win7-x86.Microsoft.NETCore.TestHost\1.0.0-beta-23413
C:\fs\visualfsharp\tests\fsharp\..\..\packages\runtime.win7-x86.Microsoft.NETCore.Windows.ApiSets\1.0.1-beta-23413
```
Perhaps it isn't the correct set of files. If I turn of the verbose option for CoreRun when executing the `fsx` tests, I get:
```
 HOSTLOG: Host directory: C:\fs\visualfsharp\tests\testbin\release\coreclr\fsharp\core\access\Compiler\
 HOSTLOG: CORE_ROOT not set; skipping
 HOSTLOG: You can set the environment variable CORE_ROOT to point to the path
 HOSTLOG: where CoreCLR.dll lives to help CoreRun.exe find it.
 HOSTLOG: Attempting to load: C:\fs\visualfsharp\tests\testbin\release\coreclr\fsharp\core\access\Compiler\CoreCLR.dll
 HOSTLOG: Loaded: C:\fs\visualfsharp\tests\testbin\release\coreclr\fsharp\core\access\Compiler\CoreCLR.dll
 HOSTLOG: Loading: C:\fs\visualfsharp\tests\testbin\release\coreclr\fsharp\core\access\Compiler\fsc.exe
 HOSTLOG: Finding GetCLRRuntimeHost(...)
 HOSTLOG: Calling GetCLRRuntimeHost(...)
 HOSTLOG: Setting ICLRRuntimeHost2 startup flags
 HOSTLOG: Starting ICLRRuntimeHost2
 HOSTLOG: Failed to start CoreCLR. ERRORCODE: 0x80004005/E_FAIL
 HOSTLOG: Execution failed
```

image

image

I noticed it is copying the `win7-x86` libraries. I'm on 64-bit Windows 10. Do I need something else? I noticed that the `Windows.ApiSets` `runtimes` has multiple subfolders that do not match the parent folder `win7-x86`:

```
PS C:\fs\visualfsharp> tree /f C:\fs\visualfsharp\packages\runtime.win7-x64.Microsoft.NETCore.Windows.ApiSets\1.0.1-beta-23413\runtimes
├───win10-x64
│   └───native
│           _._
│
├───win7-x64
│   └───native
│           API-MS-Win-Base-Util-L1-1-0.dll
│           api-ms-win-core-com-l1-1-0.dll
│           api-ms-win-core-com-private-l1-1-0.dll
│           api-ms-win-core-comm-l1-1-0.dll
│           api-ms-win-core-console-l1-1-0.dll
│           api-ms-win-core-console-l2-1-0.dll
│           api-ms-win-core-datetime-l1-1-0.dll
│           api-ms-win-core-datetime-l1-1-1.dll
│           api-ms-win-core-debug-l1-1-0.dll
│           api-ms-win-core-debug-l1-1-1.dll
│           api-ms-win-core-delayload-l1-1-0.dll
│           api-ms-win-core-errorhandling-l1-1-0.dll
│           api-ms-win-core-errorhandling-l1-1-1.dll
│           api-ms-win-core-fibers-l1-1-0.dll
│           api-ms-win-core-fibers-l1-1-1.dll
│           api-ms-win-core-file-l1-1-0.dll
│           api-ms-win-core-file-l1-2-0.dll
│           api-ms-win-core-file-l1-2-1.dll
│           api-ms-win-core-file-l2-1-0.dll
│           api-ms-win-core-file-l2-1-1.dll
│           api-ms-win-core-handle-l1-1-0.dll
│           api-ms-win-core-heap-l1-1-0.dll
│           api-ms-win-core-heap-obsolete-l1-1-0.dll
│           api-ms-win-core-interlocked-l1-1-0.dll
│           api-ms-win-core-io-l1-1-0.dll
│           api-ms-win-core-io-l1-1-1.dll
│           api-ms-win-core-kernel32-legacy-l1-1-0.dll
│           api-ms-win-core-kernel32-legacy-l1-1-1.dll
│           api-ms-win-core-kernel32-legacy-l1-1-2.dll
│           API-MS-Win-Core-Kernel32-Private-L1-1-0.dll
│           API-MS-Win-Core-Kernel32-Private-L1-1-1.dll
│           API-MS-Win-Core-Kernel32-Private-L1-1-2.dll
│           api-ms-win-core-libraryloader-l1-1-0.dll
│           api-ms-win-core-libraryloader-l1-1-1.dll
│           api-ms-win-core-localization-l1-2-0.dll
│           api-ms-win-core-localization-l1-2-1.dll
│           api-ms-win-core-localization-l2-1-0.dll
│           api-ms-win-core-localization-obsolete-l1-2-0.dll
│           api-ms-win-core-memory-l1-1-0.dll
│           api-ms-win-core-memory-l1-1-1.dll
│           api-ms-win-core-memory-l1-1-2.dll
│           api-ms-win-core-memory-l1-1-3.dll
│           api-ms-win-core-namedpipe-l1-1-0.dll
│           api-ms-win-core-namedpipe-l1-2-1.dll
│           api-ms-win-core-normalization-l1-1-0.dll
│           API-MS-Win-Core-PrivateProfile-L1-1-0.dll
│           api-ms-win-core-privateprofile-l1-1-1.dll
│           api-ms-win-core-processenvironment-l1-1-0.dll
│           api-ms-win-core-processenvironment-l1-2-0.dll
│           api-ms-win-core-processsecurity-l1-1-0.dll
│           api-ms-win-core-processthreads-l1-1-0.dll
│           api-ms-win-core-processthreads-l1-1-1.dll
│           api-ms-win-core-processthreads-l1-1-2.dll
│           API-MS-Win-Core-ProcessTopology-Obsolete-L1-1-0.dll
│           api-ms-win-core-profile-l1-1-0.dll
│           api-ms-win-core-psapi-ansi-l1-1-0.dll
│           api-ms-win-core-psapi-l1-1-0.dll
│           api-ms-win-core-psapi-obsolete-l1-1-0.dll
│           api-ms-win-core-realtime-l1-1-0.dll
│           api-ms-win-core-registry-l1-1-0.dll
│           api-ms-win-core-registry-l2-1-0.dll
│           api-ms-win-core-rtlsupport-l1-1-0.dll
│           api-ms-win-core-shlwapi-legacy-l1-1-0.dll
│           api-ms-win-core-shlwapi-obsolete-l1-1-0.dll
│           api-ms-win-core-shutdown-l1-1-0.dll
│           api-ms-win-core-shutdown-l1-1-1.dll
│           api-ms-win-core-string-l1-1-0.dll
│           API-MS-Win-Core-String-L2-1-0.dll
│           api-ms-win-core-string-obsolete-l1-1-0.dll
│           api-ms-win-core-string-obsolete-l1-1-1.dll
│           API-MS-Win-Core-StringAnsi-L1-1-0.dll
│           api-ms-win-core-stringloader-l1-1-0.dll
│           api-ms-win-core-stringloader-l1-1-1.dll
│           api-ms-win-core-synch-l1-1-0.dll
│           api-ms-win-core-synch-l1-2-0.dll
│           api-ms-win-core-sysinfo-l1-1-0.dll
│           api-ms-win-core-sysinfo-l1-2-0.dll
│           api-ms-win-core-sysinfo-l1-2-1.dll
│           api-ms-win-core-sysinfo-l1-2-2.dll
│           api-ms-win-core-sysinfo-l1-2-3.dll
│           api-ms-win-core-threadpool-l1-2-0.dll
│           api-ms-win-core-threadpool-legacy-l1-1-0.dll
│           api-ms-win-core-threadpool-private-l1-1-0.dll
│           api-ms-win-core-timezone-l1-1-0.dll
│           api-ms-win-core-url-l1-1-0.dll
│           api-ms-win-core-util-l1-1-0.dll
│           api-ms-win-core-version-l1-1-0.dll
│           api-ms-win-core-winrt-error-l1-1-0.dll
│           api-ms-win-core-winrt-error-l1-1-1.dll
│           api-ms-win-core-winrt-l1-1-0.dll
│           api-ms-win-core-winrt-registration-l1-1-0.dll
│           api-ms-win-core-winrt-robuffer-l1-1-0.dll
│           api-ms-win-core-winrt-roparameterizediid-l1-1-0.dll
│           api-ms-win-core-winrt-string-l1-1-0.dll
│           api-ms-win-core-wow64-l1-1-0.dll
│           api-ms-win-core-xstate-l1-1-0.dll
│           api-ms-win-core-xstate-l2-1-0.dll
│           API-MS-Win-devices-config-L1-1-0.dll
│           API-MS-Win-devices-config-L1-1-1.dll
│           API-MS-Win-Eventing-ClassicProvider-L1-1-0.dll
│           API-MS-Win-Eventing-Consumer-L1-1-0.dll
│           API-MS-Win-Eventing-Controller-L1-1-0.dll
│           API-MS-Win-Eventing-Legacy-L1-1-0.dll
│           API-MS-Win-Eventing-Provider-L1-1-0.dll
│           API-MS-Win-EventLog-Legacy-L1-1-0.dll
│           api-ms-win-ro-typeresolution-l1-1-0.dll
│           api-ms-win-security-base-l1-1-0.dll
│           api-ms-win-security-cpwl-l1-1-0.dll
│           api-ms-win-security-cryptoapi-l1-1-0.dll
│           api-ms-win-security-lsalookup-l2-1-0.dll
│           api-ms-win-security-lsalookup-l2-1-1.dll
│           API-MS-Win-Security-LsaPolicy-L1-1-0.dll
│           api-ms-win-security-provider-l1-1-0.dll
│           api-ms-win-security-sddl-l1-1-0.dll
│           api-ms-win-service-core-l1-1-0.dll
│           api-ms-win-service-core-l1-1-1.dll
│           api-ms-win-service-management-l1-1-0.dll
│           api-ms-win-service-management-l2-1-0.dll
│           api-ms-win-service-private-l1-1-0.dll
│           api-ms-win-service-private-l1-1-1.dll
│           api-ms-win-service-winsvc-l1-1-0.dll
│           ext-ms-win-advapi32-encryptedfile-l1-1-0.dll
│
├───win8-x64
│   └───native
│           api-ms-win-core-file-l1-2-1.dll
│           api-ms-win-core-file-l2-1-1.dll
│           api-ms-win-core-kernel32-legacy-l1-1-1.dll
│           api-ms-win-core-kernel32-legacy-l1-1-2.dll
│           API-MS-Win-Core-Kernel32-Private-L1-1-1.dll
│           API-MS-Win-Core-Kernel32-Private-L1-1-2.dll
│           api-ms-win-core-localization-l1-2-1.dll
│           api-ms-win-core-localization-obsolete-l1-2-0.dll
│           api-ms-win-core-memory-l1-1-2.dll
│           api-ms-win-core-memory-l1-1-3.dll
│           api-ms-win-core-namedpipe-l1-2-1.dll
│           api-ms-win-core-privateprofile-l1-1-1.dll
│           api-ms-win-core-processthreads-l1-1-2.dll
│           api-ms-win-core-shutdown-l1-1-1.dll
│           api-ms-win-core-string-obsolete-l1-1-1.dll
│           api-ms-win-core-stringloader-l1-1-1.dll
│           api-ms-win-core-sysinfo-l1-2-1.dll
│           api-ms-win-core-sysinfo-l1-2-2.dll
│           api-ms-win-core-sysinfo-l1-2-3.dll
│           api-ms-win-core-winrt-error-l1-1-1.dll
│           api-ms-win-core-xstate-l2-1-0.dll
│           API-MS-Win-devices-config-L1-1-1.dll
│           api-ms-win-security-cpwl-l1-1-0.dll
│           api-ms-win-security-cryptoapi-l1-1-0.dll
│           api-ms-win-security-lsalookup-l2-1-1.dll
│           api-ms-win-service-private-l1-1-1.dll
│
└───win81-x64
    └───native
            api-ms-win-core-kernel32-legacy-l1-1-2.dll
            API-MS-Win-Core-Kernel32-Private-L1-1-2.dll
            api-ms-win-core-memory-l1-1-3.dll
            api-ms-win-core-namedpipe-l1-2-1.dll
            api-ms-win-core-string-obsolete-l1-1-1.dll
            api-ms-win-core-sysinfo-l1-2-2.dll
            api-ms-win-core-sysinfo-l1-2-3.dll
            api-ms-win-security-cpwl-l1-1-0.dll
```

**Comment by ctaggart** [comment]
Closing this PR. I learned quite a bit. I'll continue experimenting on my own fork. I found `dnu` about as frustrating as `nuget` client. I'm going to see what replacing it with paket looks like. I've got FSharp.Core compiling. The rest shouldn't be too bad. https://github.com/ctaggart/visualfsharp/tree/paket

**Comment by forki** [comment]
Yeah from what I saw in dnx I don't think things really got better. If you need any help to make paket work, then just let me know.

**Pull request closed without merging**